scripts/governance.update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -ueo pipefail
+
+IC_DIR="$(realpath "$1")"
+IC_SUBDIR=rs/nns/governance/canister/governance.did
+IC_COMMIT="$(cd "${IC_DIR}" && git rev-parse HEAD)"
```

Comment:
Not sure I understand this. A `did` file is created based on a commit?

Reply:
This is for traceability.  The did file is sourced from the IC repo at that commit and I add a comment to the copied file giving its provenance.  The did compiler ignores the comment.

Reply:
ok, thanks for the explanation!